tools/import_differ/differ_utils.py

```diff
@@ -141,20 +97,23 @@ def get_gcs_data(uri: str, tmp_dir: str) -> str:
     dirname = os.path.dirname(file_pat)
```

Comment:
should `bucket = client.get_bucket(uri.split('/')[2])` be `bucket = client.get_bucket(uri.split('/')[2:])`